devices/mercator.js

```diff
@@ -30,10 +31,108 @@ module.exports = [
         },
     },
     {
-        fingerprint: tuya.fingerprint('TS011F', ['_TZ3210_7jnk7l3k', '_TZ3210_raqjcxo5']),
```

Comment:
Why was `_TZ3210_raqjcxo5` removed?

Reply:
@Koenkk 

Looks like I forgot to update when I was editing and left in _TZ3210_tfxwxklq - should be _TZ3210_raqjcxo5.

[{modelID: 'TS011F', manufacturerName: '_TZ3210_7jnk7l3k'}] is the outdoors double powerpoint, SPP02GIP
[{modelID: 'TS011F', manufacturerName: '_TZ3210_raqjcxo5'}], is the indoors double powerpoint, SPP02G

_TZ3210_tfxwxklq should be _TZ3210_raqjcxo5

Reply:
Can you update your PR?

Reply:
Done - I think? It's showing a second commit.